src/repel_boxes.cpp

```diff
@@ -1103,6 +1105,12 @@ DataFrame repel_boxes2(
     } // loop through all text labels
   } // while any overlaps exist and we haven't reached max iterations
 
+  if (verbose) {
+    if (elapsed_time > max_time)  Rprintf("%.2fs elapsed during %d iterations before completing text repel\n", max_time/1e9, iter);
+    else if (iter >= max_iter) Rprintf("%d iterations finished in %.2fs before completing text repel\n", max_iter, elapsed_time/1e9);
+    else Rprintf("text repel complete in %d iterations (%.2fs)\n", iter, elapsed_time/1e9);
```

Comment:
used `Rprintf` because that's what I'm used to, I saw some `Rcout` stuff but I don't know how to combine that with `%.2fs` / `%d`